typings/index.d.ts

```diff
@@ -0,0 +1,52 @@
+import * as React from 'react';
+import { FormHelperTextProps } from '@material-ui/core/FormHelperText'
+
+export interface ChipRendererArgs {
+  value: string;
+  text: string;
+  chip: any;
+  isFocused: boolean;
+  isDisabled: boolean;
+  handleClick: React.EventHandler<any>;
+  handleDelete: React.EventHandler<any>;
+  className: string;
+}
+
+export type ChipRenderer = (
+  args: ChipRendererArgs,
+  key: number
```

Comment:
`key` should be `any`, it's the key we get from React, we have no control over its type